### PR TITLE
Adding a test for R Translation Transform GetOffset method.

### PR DESCRIPTION
**Pull request by zivy**
The GetOffset method doesn't return the expected vector.

Change-Id: I7d152ce69955de049d94c1b6473cbc05102a0d36

### CODE REVIEW

**Comment by blowekamp** [comment]
Thank you for adding a test to demonstrate the issue with SWIG. This should be fixed in swig by swig/swig#839. Merging this patch is pending updating swig.

File: Testing/Unit/RTransformTests.R [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  tx <- c(1,2,3)
+  translation <- TranslationTransform(dimension, tx)
+  offset <- translation$GetOffset()
+  if (! (tx[1] == offset[1] && tx[2] == offset[2] && tx[3] == offset[3]))
+  {
+    cat("failure in", fname, "\n")
+    quit(save="no", status=1)
+   }
+}
+
```
Comment (blowekamp) [inline Testing/Unit/RTransformTests.R]: Wasn't there another test in the gerrit patch set?

Comment (zivy) [inline Testing/Unit/RTransformTests.R]: Yes, I just cherry picked it off of gerrit and pushed to this branch. Branch name "swigOffsetBug" is not correct for both commits but the commit messages describe the problems the tests address.

**Comment by blowekamp** [comment]
topic merged into next

**Pull request merged**